probe/kubernetes/reporter.go

```diff
@@ -149,6 +153,18 @@ var (
 		Storage:           {ID: Storage, Label: "Capacity", From: report.FromLatest, Priority: 9},
 	}
 
+	StoragePoolMetadataTemplates = report.MetadataTemplates{
+		NodeType:   {ID: NodeType, Label: "Type", From: report.FromLatest, Priority: 1},
+		Name:       {ID: Name, Label: "Name", From: report.FromLatest, Priority: 2},
```

Comment:
Name is not required here because name is already shown on top, it's not a part of metadata.